app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -253,6 +253,7 @@ private function getOrCreateUser($record, $values)
                 'addr_state' => $record['Home state'],
                 'addr_zip' => $record['Home zip code'],
                 'source' => env('NORTHSTAR_CLIENT_ID'),
+                'email_subscription_status' => $this->transformEmailSubscriptionStatus($record['Opt-in to Partner email?'])
```

Comment:
Can we always assume this column value will be sent, or should we conditionally check whether the column exists before setting? cc @ngjo

Reply:
The column *should* always be sent as long as RTV doesn't change their export settings or the name of the column! It hasn't changed in the past year that we've been exporting/ingesting so there's likely only a slim chance that that column wouldn't be there. If it isn't there for whatever reason, would it just break the import or would everyone get subscribed or something?

Reply:
As is, everyone would have their `email_subscription_status` set to false (assuming I figure out how to set it to false per question I flagged in the PR description)

Reply:
Updated this so we only update the value if the column exists.